client/src/modules/list/components/Items/model/actionTypes.js

```diff
@@ -8,6 +8,7 @@ export const CommentActionTypes = enumerable('comment')(
 export const ItemActionTypes = enumerable('item')(
   ...asyncTypes('ADD'),
   ...asyncTypes('ARCHIVE'),
+  'DISABLE_ANIMATIONS',
```

Comment:
after refactoring actions (check next comments) you will need success/ failure here
so remember to refactor to 
`...asyncTypes('DISABLE_ANIMATIONS'),`